frontend/components/Button/index.css

```diff
@@ -2,14 +2,60 @@
 @import '../../styles/colors.css';
 @import '../../styles/layout.css';
 
+$button-offset: 4px;
+
 .root {
-  display: flex;
+  position: relative;
+
+  a,
```

Comment:
You can give that element a class and avoid this, right?

Reply:
Yeah